README.rst

```diff
@@ -59,3 +59,20 @@ The underlying JSON implementation can be chosen with the following:
     which uses the standard library json module).
 
 .. _simplejson: https://simplejson.readthedocs.io/
+
+A preserialisation hook allows you to encode objects which aren't encodable by the
+standard library ``JSONEncoder``.
```

Comment:
Did you want to update the version or changelog in this PR or will do it separate?

Reply:
Was going to leave it---I think we usually make bumping the version number part of a release